src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsnonassociationsapi/jpa/NonAssociation.kt

```diff
@@ -57,4 +58,20 @@ class NonAssociation(
     this.closedReason = closedReason
     this.closedAt = closedAt
   }
+
+  fun toDto(): NonAssociationDTO {
+    return NonAssociationDTO(
+      id = id!!,
+      firstPrisonerNumber = firstPrisonerNumber,
+      firstPrisonerReason = firstPrisonerReason,
+      secondPrisonerNumber = secondPrisonerNumber,
+      secondPrisonerReason = secondPrisonerReason,
+      restrictionType = restrictionType,
+      comment = comment,
+      // TODO: Do we need to do anything special with this?
+      //       This field being optional in NOMIS/Prison API
+      //       It may be one of the things we make mandatory after migration?
+      authorisedBy = authorisedBy ?: "",
```

Comment:
This isn't a field at the moment in the design I believe.

Reply:
I've named this `authorisedBy` as it's called in Prison API/NOMIS.
It seems to be free text and optional in Prison API but for records created via Non-associations API this field would be populated with the username of the user who created it.